bin/cli.js

```diff
@@ -131,8 +79,8 @@ yargs
   })
   .fail((err) => {
     // Prints to `stderr` by default
-    return showErrorAndQuit(err);
+    return helpers.showErrorAndQuit(err);
   })
   .argv;
 
-process.on('uncaughtException', showErrorAndQuit);
+process.on('uncaughtException', helpers.showErrorAndQuit);
```

Comment:
It's worth adding a case here to catch 'unhandledRejection' too, so promise errors end up here as well.